### ISSUE TITLE
How to run the project locally?

### ISSUE THREAD

**Issue by nadav96** [hjalmers/angular-generic-table#83]
Hi, I'm trying to set up the project locally, but when I run npm install (which after install call `npm run -s build`), fail with the following error:
`Error File 'C:/Users/Nadav/Desktop/gt/src/index.ts' not found.
`

The script in the package.json uses tsconfig-aot.json as the config file, which upon inspection seems to have a reference to unknown file at src directory.
```
  "files": [
    "index.ts"
  ],
```

Tried also to link the project as npm package using the following package.json line:
`"angular-generic-table": "file:/path/to/local/angular-generic-table"`

But it fails on the same issue..

Can you please explain how to run the project locally, as I would like to fork it and contribute.

Thanks.

**Comment by hjalmers** [comment]
Yeah currently it's not super obvious and the project and setup could use some tidying since we went for scoped packages. But in the meantime you should be able to ignore the build error, the project is based on angular-cli and so long as all the dependencies were installed you should be able to start the project locally by running `ng serve`.

**Comment by hjalmers** [comment]
Hi @nadav96, were you able to run the project locally? I've edited the tsconfig file now so it should be a bit easier to get everything up and running. Basically you should just need to fork/clone the repo and run npm install followed by ng serve.
